packages/commonwealth/test/integration/api/external/getThreads.spec.ts

```diff
@@ -134,4 +133,30 @@ describe('getThreads Tests', () => {
     chai.assert.equal(resp.result[0].msg, 'Invalid value');
     chai.assert.equal(resp.result[0].param, 'count_only');
   });
+
+  it('Should return sensible default for thread with no associated comment', async () => {
+    let resp = await get(
+        '/api/threads',
+        { community_id: testThreads[0].chain, include_comments: true, topic_id: -1 }
+      );
+
+    chai.assert.lengthOf(resp.result.threads, 2);
+    if (resp.result.threads[0].Comments.length === 2){
+      chai.assert.equal(resp.result.threads[0].Comments.length, 2);
+      chai.assert.equal(resp.result.threads[1].Comments.length, 3);
+    } else {
+      chai.assert.equal(resp.result.threads[0].Comments.length, 3);
+      chai.assert.equal(resp.result.threads[1].Comments.length, 2);
+    }
+
+    resp = await get(
+      '/api/threads',
+      { community_id: testThreads[0].chain, include_comments: true, topic_id: -2 }
```

Comment:
may be add test `include_comments` false - `comments` attribute should not be there in response